main.c

```diff
@@ -321,11 +328,13 @@ void setZombieTextTexture(ENEMY *block) {
   SDL_FreeSurface(textSurface);
 }
 
+#if !defined(DISABLE_AUDIO)
```

Comment:
nit: As alternative implementation you could probably have left all `playSound` calls enabled and just disabled the sound backend by conditionally removing the contents of this function (but not removing this function itself). This would have required something like `typedef void* Mix_Chunk; /* Dummy */` instead of including SDL_mixer.